yjit/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+# NOTE: please avoid adding dependencies to external crates as these can
+# make building and packaging YJIT more challenging.
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[package]
+name = "yjit"
+version = "0.1.0"       # YJIT version
+edition = "2021"        # Rust 2021 edition to compile with
+rust-version = "1.57.0" # Minimally supported rust version
+publish = false         # Don't publish to crates.io
+
+[lib]
+crate-type = ["staticlib"]
+
+[dependencies]
+# No required dependencies to simplify build process. TODO: Link to yet to be
+# written rationale. Optional For development and testing purposes
+# FIXME: `capstone-sys` takes 23s to build in dev mode and 50s in release mode.
+# It might be better to dynamically link against system capstone instead of
+# depending on this crate.
+capstone = { version = "0.10.0", optional = true }
+
+[features]
+# NOTE: By having the Capstone dependency behind a feature, we allow
+# building without needing to install it, but then I have to run
+# `cargo test --features disassembly` for tests...
```

Comment:
nit: Probably a typo, as the feature below is called `disasm`, not `disassembly`